tools/digester/main.go

```diff
@@ -274,13 +276,29 @@ func readOneDigest(ctx context.Context, image *Image, index int, wg *sync.WaitGr
 	imgRef, err := docker.ParseReference(fullName)
 	exitOnError(err, "failed to parse container reference")
 
-	digest, err := docker.GetDigest(ctx, nil, imgRef)
+	digest, err := retryGetDigest(5, 1, ctx, nil, imgRef)
```

Comment:
Duration of 1 is one nanosecond - isn't it too fast? maybe start from 1 second? Just need to make sure that the final duration won't cause timeout.

Reply:
Now it can potentially range from:
0, 1, 3, 9, 27 = 40 seconds up to:
0, 1+0.5, 3+1.5, 9+4.5, 27+13.5 = 60 seconds